### PR TITLE
feat(signin): handle new account must reset error (126)

### PR DESCRIPTION
**Pull request by shane-tomlinson**
display an error message on signin when the user must reset with a link to /confirm_password_reset

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@dannycoates, @philbooth and @vbudhram - mind an r? I extracted @dannycoates's original logic into a mixin that can be shared by several views.

File: app/scripts/views/mixins/account-reset-mixin.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  var AccountResetMixin = {
+    initialize: function (options) {
+      options = options || {};
+
+      this._session = options.session;
+    },
+
+    events: {
+      'click a[href="/confirm_reset_password"]':
+            BaseView.preventDefaultThen('sendAccountResetEmail')
```
Comment (shane-tomlinson) [inline app/scripts/views/mixins/account-reset-mixin.js]: nit: indentation

**Comment by philbooth** [comment]
LGTM :+1:

File: app/tests/spec/views/sign_in.js [hunk 2]
```diff
@@ -252,6 +252,25 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('with a reset accont', function () {
```
Comment (vbudhram) [inline app/tests/spec/views/sign_in.js]: nit: `account`

**Comment by vbudhram** [comment]
Besides nit, LGTM

**Comment by dannycoates** [comment]
magic! :tophat: :rabbit:

**Pull request merged**